packages/myst-frontmatter/src/project/validators.ts

```diff
@@ -34,7 +34,7 @@ export function validateProjectAndPageFrontmatterKeys(
 ) {
   const output: ProjectAndPageFrontmatter = validateSiteFrontmatterKeys(value, opts);
   if (defined(value.date)) {
-    output.date = validateDate(value.date, incrementOptions('date', opts));
+    output.date = validateDate(value.date, { ...incrementOptions('date', opts), dateIsUTC: true });
```

Comment:
This encodes the assumption that `Date` objects passed in to the validator should be treated as plain dates in UTC (i.e. take components in UTC).